src/components/03-components/01-actions/actions.config.yml

```diff
@@ -10,8 +10,23 @@ context:
         dataTransform: grow-5
     layeredIcon:
         type: layers
+        classes:  padding-left-4
         icons:
             -   icon: circle
                 prefix: fas
+                classes: text-primary
+                dataTransform: grow-8
             -   icon: ellipsis
-                dataTransform: shrink-6
+                classes: text-secondary-dark
+                dataTransform: shrink-1
+    layeredIcon2:
+        type: layers
+        classes:  padding-left-4
+        icons:
+            -   icon: circle
+                prefix: fas
+                classes: text-primary
+                dataTransform: grow-12
+            -   icon: ellipsis
+                classes: text-secondary-dark
```

Comment:
text-secondary-dark is redundant since it's already being used in button class so will automatically inherit as discussed yesterday

Reply:
done